docs/en/integrations/data-ingestion/data-formats/parquet-arrow-avro-orc.md

```diff
@@ -0,0 +1,325 @@
+---
+sidebar_label: Parquet, Avro, Arrow and ORC
+sidebar_position: 4
+slug: /en/integrations/data-formats/parquet-arrow-avro-orc
+---
+
+# Working with Parquet, Avro, Arrow, and ORC data in ClickHouse
+
+Apache has released multiple data formats actively used in analytics environments, including the most popular [Parquet](https://parquet.apache.org/), [Avro](https://avro.apache.org/), [Arrow](https://arrow.apache.org/), and [Orc](https://orc.apache.org/). ClickHouse supports importing and exporting data using any from that list.
+
+## Working with Parquet data
+
+Parquet is an efficient file format to store data in a column-oriented way.
+
+### Importing from Parquet
+
+Before loading data, we can use [file()](/docs/en/sql-reference/functions/files.md/#file) function to explore an [example parquet file](assets/data.parquet) structure:
+
+```sql
+DESCRIBE TABLE file('data.parquet', Parquet)
+```
+
+:::tip
+When using the `file()` function, with ClickHouse Cloud you will need to run the commands in `clickhouse client` on the machine where the file resides.
```

Comment:
It's confusing. `file()` table function reads files from the machine on which server is running, not client.

Reply:
Already pushed and deployed - in each article where we have `file()` function call, it now has a [tip block](https://clickhouse.com/docs/en/integrations/data-formats/parquet-arrow-avro-orc#importing-from-parquet).

![image](https://user-images.githubusercontent.com/85636145/214822168-1da36073-aca3-4102-9457-763853045319.png)